src/styles/blocks/button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+  min-width: 166px;
+  min-height: 40px;
+  border-radius: 5px;
+  padding: auto;
+  margin: 16px;
+  align-items: center;
+  background: #00acdc;
+  border: 0;
+  justify-content: center;
+  display: flex;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-decoration: none;
+
+  &:hover {
```

Comment:
The '&:hover' selector should include 'cursor: pointer;' to indicate that the button is clickable.